packages/project-access/src/project/cap.ts

```diff
@@ -295,3 +319,52 @@ async function readPackageNameForFolder(baseUri: string, relativeUri: string): P
     }
     return packageName;
 }
+
+let globalCdsPathCache: string;
+
+/**
+ * Try to load global installation of @sap/cds, usually child of @sap/cds-dk.
+ *
+ * @returns - module @sap/cds from global installed @sap/cds-dk
+ */
+async function loadGlobalCdsModule<T>(): Promise<T> {
+    if (!globalCdsPathCache) {
+        const versions = await getCdsVersionInfo();
+        if (!versions.home) {
+            throw Error('Can not find global installation of module @sap/cds, which should be part of @sap/cds-dk');
+        }
+        globalCdsPathCache = versions.home;
+    }
+    return loadModuleFromProject<T>(globalCdsPathCache, '@sap/cds');
+}
+
+/**
+ * Get cds information, which includes versions and also the home path of cds module.
+ *
+ * @param [cwd] - optional folder in which cds --version should be executed
+ * @returns - result of call 'cds --version'
+ */
+function getCdsVersionInfo(cwd?: string): Promise<Record<string, string>> {
```

Comment:
Adding async signature to the function?

Reply:
Will be addressed in a separate PR, thanks @slin-sap